e2e/helpers/puppeter_helper.js

```diff
@@ -9,6 +9,11 @@ module.exports = class PuppeteerHelpers extends Helper {
     return page.reload();
   }
 
+  waitForNavigationTest() {
```

Comment:
Is this used? I don't see any changes invoking this method.

Reply:
the last of other options tried, removed